_includes/releases/v23.1/v23.1.0-alpha.4.md

```diff
@@ -6,17 +6,17 @@ Release Date: February 27, 2023
+- Deprecate and replace some of the changefeed specific functions made available in preview mode in 22.2 release.   While these functions are deprecated, old changefeed transformations should continue to function properly. Customers are encouraged to closely monitor their changefeeds created during upgrade. While effort was made to maintain backward compatibility, the updated changefeed transformation may produce slightly different output (different column names, etc). [#96295][#96295]
+- Add support for implicit authentication to Azure Storage and KMS. [#96825][#96825]
+- Add support for `CREATE EXTERNAL CONNECTION ... AS "postgresql://"` or `"postgres://"`. These external connections may be specified as the source in streaming replication. [#96551][#96551]
+- Add support for referencing [user defined functions (UDFs)](../{{site.versions["stable"]}}/user-defined-functions.html) from other objects. Backup and restore operations can now read and write UDF descriptors. [#97038][#97038]
 
 <h3 id="v23-1-0-alpha-4-sql-language-changes">SQL language changes</h3>
 
-- Set-returning user-defined functions can now be created by specific a `SETOF` return type. [#96698][#96698]
+- UDFs can now return a set of results by setting the return type to `SETOF`. [#96698][#96698]
 - UDFs with implicit record return types will return an error when called if the return type has been altered and is no longer compatible with the body of the UDF. [#96696][#96696]
 - Implemented the `COPY ... TO STDOUT` statement, which allows exporting a table or arbitrary query using the postgres wire-compatible format. Only text & CSV format is supported. [#94408][#94408]
-- Added the read-only "ssl" session variable. It returns "off" if the server was started in insecure mode (which disables TLS), or "on" otherwise. This is based on the Postgres variable of the same name. [#97257][#97257]
+- Add the read-only "ssl" session variable. It returns "off" if the server was started in insecure mode (which disables TLS), or "on" otherwise. This is based on the Postgres variable of the same name. [#97257][#97257]
```

Comment:
I don't think a session variable "returns" anything -- reword that it is set to "off" or "on"?